frontend/src/scenes/saved-insights/savedInsightsLogic.ts

```diff
@@ -251,7 +251,8 @@ export const savedInsightsLogic = kea<savedInsightsLogicType>({
         duplicateInsight: async ({ insight, redirectToInsight }) => {
             insight.name = (insight.name || insight.derived_name) + ' (copy)'
             const newInsight = await api.create(`api/projects/${values.currentTeamId}/insights`, insight)
-            actions.loadInsights()
+            values.insights.results.push(newInsight)
```

Comment:
Can we make this an action/reducer?

We should think of `values` as immutable and redux as functional programming. Doing otherwise abstractions around listeners and so on just break down and code becomes harder to grok since you're dealing with two different layers of abstraction.